src/main/java/org/folio/service/job/JobExecutionServiceImpl.java

```diff
@@ -200,6 +214,24 @@ private void prepareJobExecution(JobExecution jobExecution, FileDefinition fileE
     return jobExecutionPromise.future();
   }
 
+  private void updateErrorLogIfJobIsExpired(String jobExecutionId, String tenantId) {
+    Future<List<ErrorLog>> listFuture = errorLogService.getByQuery(HelperUtils.getErrorLogCriterionByJobExecutionId(jobExecutionId), tenantId);
+    if (listFuture != null) {
+      listFuture.onSuccess(errorLogs -> {
+        if (!errorLogs.isEmpty()) {
+          ErrorLog errorLog = errorLogs.get(0);
+          errorLog.setErrorMessageCode(ErrorCode.ERROR_JOB_IS_EXPIRED.getCode());
+          errorLog.setErrorMessageValues(List.of(ErrorCode.ERROR_JOB_IS_EXPIRED.getDescription()));
```

Comment:
Please use a chain of .withXXX() methods instead of setters

Reply:
Fixed, thank you.